src/rexi/src/rexi_utils.erl

```diff
@@ -95,6 +107,49 @@ process_message(RefList, Keypos, Fun, Acc0, TimeoutRef, PerMsgTO) ->
             end;
         {rexi_DOWN, _, _, _} = Msg ->
             Fun(Msg, nil, Acc0)
-    after PerMsgTO ->
-        {timeout, Acc0}
+    end.
+
+add_delta({A}, Delta) -> {A, Delta};
+add_delta({A, B}, Delta) -> {A, B, Delta};
+add_delta({A, B, C}, Delta) -> {A, B, C, Delta};
+add_delta({A, B, C, D}, Delta) -> {A, B, C, D, Delta};
+add_delta({A, B, C, D, E}, Delta) -> {A, B, C, D, E, Delta};
+add_delta({A, B, C, D, E, F}, Delta) -> {A, B, C, D, E, F, Delta};
+add_delta({A, B, C, D, E, F, G}, Delta) -> {A, B, C, D, E, F, G, Delta};
```

Comment:
At first sight this looks a bit awkward. Do we have any `add_delta({A, B, C, D, E, F}, Delta) -> {A, B, C, D, E, F, Delta};` message sent to rexi?

It might be better to be explicit about what message rexi will actually get. We may have to do a two-stage PRs. One to prep the receivers to accept both old and new patterns, then another one to start sending the new messages.